uw_sws/notice.py

```diff
@@ -48,8 +48,7 @@ def _notices_from_json(notice_data):
                 elif attribute.data_type == "date":
                     # Convert to UTC datetime
                     date = parser.parse(notice_attrib.get("Value"))
-                    localtz = pytz.timezone('America/Los_Angeles')
-                    local_dt = localtz.localize(date)
+                    local_dt = SWS_TIMEZONE.localize(date)
                     utc_dt = local_dt.astimezone(pytz.utc)
 
                     attribute._date_value = utc_dt
```

Comment:
You can simplify this to:

from datetime import timezone

date = parser.parse(notice_attrib.get("Value"))
attribute._date_value = date.replace(tzinfo=timezone.utc)